---
layout: page
description: I am a tenure-track Assistant Professor of Bioinformatics and Computational Biology at Wake Forest University School of Medicine.
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="https://scholar.google.com/citations?user=wIYviKIAAAAJ&hl=zh-CN&oi=ao">Google Scholar</a></li>
        <li><a href="https://profiles.wakehealth.edu/display/Person/qsong">Official Website</a></li>
      </ul>
  </div>
</div>



###  Biography
<div class="container">
    <div class="row-fluid">
        <div class="span5">
            Song Qianqian<br/>
            <a href="https://profiles.wakehealth.edu/display/Person/qsong">Assistant Professor of Bioinformatics and Computational Biology</a><br/>
		<a href="https://school.wakehealth.edu/departments/cancer-biology">Department of Cancer Biology</a><br/>
		 <a href="https://school.wakehealth.edu/">Wake Forest University School of Medicine</a><br/>
Bowman Gray Center for Medical Education, 475 Vine St <br/>
    Winston-Salem, NC 27101<br/>
            USA<br/>
            <div id="hide_email">
            Email: <code>qsong@wakehealth.edu</code><br/>
            </div>
        </div>,
</div>
<br/>
	</div>


I am a tenure-track Assistant Professor of Bioinformatics and Computational at Wake Forest University School of Medicine. I received my Ph.D degree in both [Bioinformatics](https://school.wakehealth.edu/research/institutes-and-centers/center-for-biomedical-informatics) from Wake Forest University School of Medicine, and [Mathematics](http://math.english.bnu.edu.cn/) from Beijing Normal University in 2017.


a MS in [Biostatistics](http://www.jhsph.edu/departments/biostatistics/index.html) and another MS in [Computer Science](https://www.cs.jhu.edu/) from the Johns Hopkins University in 2011, and PhD in [Computer Science and Informatics](http://www.cs.emory.edu/) from Emory University in 2017.

### Opening
My lab now has multiple openings for PhD/MS students and one opening for Postdoc. Interested candidates please refer the [Opening](https://lichen-lab.github.io//pages/opening.html) for how to apply.

### Research
My lab is currently funded by NIH NIGMS, Showalter Trust Research Fund and Indiana University Melvin and Bren Simon Comprehensive Cancer Center. My lab's research focuses on developing statistical and informatics methods for analyzing of multi-omics data. Projects include but not limited to 

- **Epigenomics**: 
Develop statistical methods and database for analyzing epigenomic data
	- Differential binding. [<a href="https://academic.oup.com/bioinformatics/article/31/12/1889/214643">ChIPComp</a>]
	- TF regulatory module. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz975/5695705">tfLDA</a>]
	- Database and web server. [<a href="https://academic.oup.com/bioinformatics/article/27/10/1447/260530">hmChIP</a>]

- **Genetics**:
Develop computational methods for annotating and analyzing functional variants (GWAS SNPs, cis-eQTL SNVs) by utilizing large-scale multi-omics profiles
	- GWAS variant enrichment analysis. [<a href="https://academic.oup.com/bioinformatics/article/32/8/1214/1744655">traseR</a>]
	- Functional variant prediction. [<a href="https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1112-z">DIVAN</a>; <a href="https://academic.oup.com/bioinformatics/article/35/9/1573/5126236">TIVAN</a>; 
	 <a href="https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbab189/6279833?redirectedFrom=fulltext">WEVar</a>
	 <a href="https://academic.oup.com/bioinformatics/advance-article-abstract/doi/10.1093/bioinformatics/btac214/6564688?redirectedFrom=fulltext">TLVar</a>]

- **Metagenomics**:
Develop statistical methods for analyzing microbiome data
	- Normalization. [<a href="https://peerj.com/articles/4600/">GMPR</a>]
	- Power analysis. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btaa197/5809526">powmic</a>]
	- Predictive modeling.[<a href="https://academic.oup.com/bioinformatics/article/31/24/3991/197681">glmgraph</a>; <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.03112/full">SICS</a>; <a href="https://www.frontiersin.org/articles/10.3389/fmicb.2018.01391/full">glmmTree</a>;
	<a href="https://academic.oup.com/bib/advance-article-abstract/doi/10.1093/bib/bbaa073/5835556?redirectedFrom=fulltext">MDeep</a>]
	
- **Single-cell genomics**:
Develop statistical methods for clustering single-cell data
	- Clustering scRNA data.[<a href="https://www.nature.com/articles/s41467-019-09639-3">BAMMSC</a>]

- **Transcriptomics**:
Develop statistical methods for analyzing non-coding RNAs
	- Differential expression analysis. [<a href="https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz606/5543088">circMeta</a>]


