---
layout: page
title: About
description: I am a tenure-tracked Assistant Professor in Harrison School of Pharmacy at Auburn University. I am a member of 
the Omics and Informatics Cluster hire in Auburn University.
---

<div class="navbar">
  <div class="navbar-inner">
      <ul class="nav">
          <li><a href="https://scholar.google.com/citations?user=Nk-yRrcAAAAJ&hl=en">Google Scholar</a></li>
        <li><a href="http://www.auburn.edu/academic/pharmacy/directory/li-chen.html">Official Website</a></li>
      </ul>
  </div>
</div>

###  Li's Biography

I am a Tenure-Track Assistant Professor in Harrison School of Pharmacy at Auburn University. I am a member of 
The Omics and Informatics Cluster Hire in Auburn University. I received a BS from China majoring in Bioinformatics in 2009; a
MHS in [Biostatistics](http://www.jhsph.edu/departments/biostatistics/index.html) in 2011, from the
Johns Hopkins University; another MSE in [Computer Science](https://www.cs.jhu.edu/) in 2011, from the
Johns Hopkins University. I joined the department right after obtaining my PhD in [Computer Science and Informatics](http://www.mathcs.emory.edu/site/home/) at Emory University in 2017.
I also interned in CareerBuilder Data Science and Amazon Machine Learning in Summer 2014 and 2015 respectively.


### Research
I mainly work on Bioinformatics and Computational Biology. To be specific, my research focuses on the development of statistical and informatics methods for the analysis of high-throughput genetic/genomic, epigenomics and other omics data,
summarized in the following areas.

- **Epigenomics**: 
Develop statistical methods and database for epigenomic data in differential biological conditions.
	- [Differential binding](https://academic.oup.com/bioinformatics/article-lookup/doi/10.1093/bioinformatics/btv094). [<a href="http://bioconductor.org/packages/devel/bioc/html/ChIPComp.html">Link to ChIPComp</a>]
	- [Database and web server](https://academic.oup.com/bioinformatics/article/27/10/1447/260530). [<a href="http://jilab.biostat.jhsph.edu/database/cgi-bin/hmChIP.pl">Link to hmChIP</a>]

- **Genetics**:
Develop computational methods for annotating and predicting functional variants (GWAS SNPs, cis-eQTL SNVs) by utilizing large-scale multi-omics profiles. 
	- [GWAS variant enrichment analysis](https://academic.oup.com/bioinformatics/article/32/8/1214/1744655). [<a href="http://bioconductor.org/packages/release/bioc/html/traseR.html">traseR</a>]
	- [Functional variant prediction](https://genomebiology.biomedcentral.com/articles/10.1186/s13059-016-1112-z). [<a href="https://sites.google.com/site/emorydivan/">DIVAN</a>]; (https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/bty872/5126236). [<a href="https://github.com/lichen-lab/TIVAN">TIVAN</a>]

- **Metagenomics**:
Develop statistical methods for predictive modeling disease status based on 16S rRNA sequencing microbiome data.
	- [Normalization](https://peerj.com/articles/4600/). [<a href="https://github.com/lichen-lab/GMPR">GMPR</a>]
	- [Power analysis](https://peerj.com/articles/4600/). [<a href="https://github.com/lichen-lab/powmic">powmic</a>]
	- [Predictive modeling](https://academic.oup.com/bioinformatics/article/31/24/3991/197681). [<a href="https://cran.r-project.org/web/packages/glmgraph/index.html">glmgraph</a>]
; (https://www.frontiersin.org/articles/10.3389/fmicb.2018.03112/abstract). [<a href="https://github.com/lichen-lab/SICS">SICS</a>]; (https://www.frontiersin.org/articles/10.3389/fmicb.2018.01391/abstract). [<a href="https://github.com/lichen-lab/glmmTree">glmmTree</a>]
	
- **Single-cell genomics**:
Develop statistical methods for clustering single cell  data.
	- [Clustering droplet-based single cell transcriptomic data from population studies](https://nature-research-under-consideration.nature.com/users/37265-nature-communications/posts/38057-bamm-sc-a-bayesian-mixture-model-for-clustering-droplet-based-single-cell-transcriptomic-data-from-population-studies). [<a href="https://github.com/lichen-lab/BAMMSC">BAMMSC</a>]

- **Transcriptomics**:
Develop statistical methods for non-coding RNA.
	- [Genomic feature annotation and differential expression analysis of circular RNAs](https://academic.oup.com/bioinformatics/advance-article/doi/10.1093/bioinformatics/btz606/5543088). [<a href="https://github.com/lichen-lab/circMeta">circMeta</a>]



### Contact
<div class="container">
    <div class="row-fluid">
        <div class="span5">
            Li Chen<br/>
            <a href="http://www.auburn.edu/academic/pharmacy/horp/faculty.html">Department: Health Outcomes Research and Policy</a><br/>
            <a href="http://www.auburn.edu">Auburn University</a><br/>
            <a href="http://www.auburn.edu/academic/pharmacy/">Harrison School of Pharmacy</a><br/>
            4306f Walker Building <br/>
            Auburn, AL 36849 <br/>
            USA<br/><br/>
            <div id="hide_email">
            Email: <code>li.chen@auburn.edu</code><br/>
            </div>
        </div>

        <div class="span2">
        <a href="../assets/pics/Chen.jpg">
            <img src="../assets/pics/Chen.jpg" height="200" width="150" title="Li Chen" alt="Li Chen"/>
        </a>
        </div>
    </div>
</div>


